.github/workflows/coding.yml

```
@@ -0,0 +1,55 @@
+
+on:
+  push:
+    branches: [ "coding*", "scala*" ]
+
+jobs:
+  inspect-code:
+    name: '[Required] Inspect code'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
```

yamllint

[warning] too few spaces before comment

(comments)